custom/enikshay/tasks.py

```diff
@@ -147,11 +195,14 @@ def update_single_case(self, episode_case):
         if update_json:
             update_case(self.domain, episode_case.case_id, update_json)
 
-    def _get_open_episode_cases(self):
+    def _get_case_ids(self):
         case_accessor = CaseAccessors(self.domain)
         case_ids = case_accessor.get_open_case_ids_in_domain_by_type(CASE_TYPE_EPISODE)
```

Comment:
How would that work? We also fetch related cases during the different tasks (which I think aren't guaranteed to be on the same partition), so not sure if we would get any gains from doing that?